catalyst_voices/Earthfile

```diff
@@ -20,38 +19,41 @@ builder:
     DO flutter-ci+BOOTSTRAP
 
 # Generates flutter code.
-# Based on Catalyst Gateway OpenAPI specifications generates models, clients
-# and serialization logic.
+# Generates codes for Catalyst Gateway OpenAPI, Voices Localization and
+# VoicesAssets and other packages that depend on code-generator.
 # It accepts [save_locally] ARG that when true place the artifacts in the
-# proper folder of `catalyst_voices_services` local code.
+# proper folders 
 code-generator:
     ARG save_locally=false
-
+    FROM +builder
     LET gen_code_path  = lib/generated/catalyst_gateway
     LET local_gen_code_path = packages/internal/catalyst_voices_services/lib/generated/catalyst_gateway/
 
-    FROM +builder
+    RUN melos l10n
+    RUN melos build_runner
+
+    IF [ $save_locally = true ]
+        RUN find . \( -name "*.g.dart" -o -name "*.freezed.dart" -o -name "*.gen.dart" -o -name "catalyst_voices_localizations*.dart" \)
+
+        FOR generated_file IN $(find . \( -name "*.g.dart" -o -name "*.freezed.dart" -o -name "*.gen.dart" -o -name "catalyst_voices_localizations*.dart" \))
```

Comment:
Make sure these file extensions are synced with the .gitignore here:

![Screenshot 2024-11-14 at 10 57 10](https://github.com/user-attachments/assets/62263127-5c32-45ba-80e5-a30c7c04d504)

Reply:
Good catch!. Thank you!